feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -388,38 +388,58 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		gnmi.Replace(t, dut, gnmi.OC().Qos().Config(), q)
 	}
 
+	var q_BE1, q_BE0, q_AF1, q_AF2, q_AF3, q_AF4, q_NC1 string
```

Comment:
underscores are not allowed in go variable name style. I'd probably create an inline type here:

	type qosVals struct {
		be0, be1, af1, af2, af3, af4, nc1 string
	}
	qos := qosVals {
		be0: "BE0",
		be1: "BE1",
		af1: "AF1",
		af2: "AF2",
		af3: "AF3",
		af4: "AF4",
		nc1: "NC1",
	}
	if dut.Vendor() == ondatra.JUNIPER {
		qos = qosVals {
			be0: "7",
			be1: "0",
			af1: "4",
			af2: "1",
			af3: "5",
			af4: "2",
			nc1: "3",
		}
	}